lib/elixir/lib/enum.ex

```diff
@@ -2129,17 +2132,22 @@ defmodule Enum do
   end
 
   @doc """
-  Returns a subset list of the given enumerable, from `range.first` to `range.last` positions.
+  Returns a subset list of the given enumerable, from `index_range.first` (zero-based) to
+  `index_range.last`.
 
-  Given `enumerable`, it drops elements until element position `range.first`,
-  then takes elements until element position `range.last` (inclusive).
+  `index_range` must be a `(range)[t:Range.t/0]`.
```

Comment:
Why `range` in backticks? I think it would read better as "`index_range` must be a range"` (with "range" being a link).

Reply:
done
    `index_range` must be a [range](`t:Range.t/0`).